src/PasswordGenerator.test.js

```diff
@@ -14,4 +14,17 @@ describe('PasswordGenerator', () => {
         expect(new Set(pws).size).toBe(1)
         expect(pwg.generated).toBe(true)
     })
+
+    // Generate 10000 passwords and ensure they all include a number.
+    // Fixes:
+    // * https://app.asana.com/0/1198964220583541/1205924276944158/f
+    // * https://app.asana.com/0/1200930669568058/1205489446428548/f
```

Comment:
I'd remove the references to the tasks, otherwise in a few weeks we'd double the repo size, lol 😅.